modules/schema/src/test/java/org/apache/ignite/internal/schema/registry/SchemaRegistryImplTest.java

```diff
@@ -35,122 +40,117 @@
 import java.util.Arrays;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
-import java.util.stream.Collectors;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
 import org.apache.ignite.internal.schema.Column;
 import org.apache.ignite.internal.schema.SchemaDescriptor;
 import org.apache.ignite.internal.schema.SchemaUtils;
 import org.apache.ignite.internal.schema.mapping.ColumnMapper;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 /**
  * Schema manager test.
  */
 public class SchemaRegistryImplTest {
+    private final SchemaDescriptor schemaV1 = new SchemaDescriptor(1,
```

Comment:
I think the field can be static.

Reply:
What's the profit?

Reply:
I have only one argument, this field looks like a constant, I don’t think it needs to be initialized every time. This question is at your discretion.